src/components/Dao.tsx

```diff
@@ -0,0 +1,18 @@
+import { Button } from '@chakra-ui/react'
+import { useRouter } from 'next/router'
+import useDaoMetadata from '@/services/useDaoMetadata'
+import { useEffect } from 'react'
+
+const AppIndex = (): JSX.Element => {
+  const router = useRouter()
+  const { id } = router.query
+  const { data, fetch } = useDaoMetadata(id)
```

Comment:
Why expose fetch? 

I think you can fetch it directly in the use... Hook. Just expose data, loading, error.